index.html

```diff
@@ -0,0 +1,190 @@
+          <a href="#"> English </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> My Page </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> Logout </a>
+        </li>
+      </ul>
+
+    </header>
+    <header class="header2">
+      <div class="menu-bar-container">
+        <ul class="menu-bar">
+          <li>
+           <a href="#" class="hamburger">
+            <i class="fa fa-bars menu-bar-icon"></i>
+           </a>
+          </li>
+          <li>
+            <a href="#" class="close-icon"> <i class="fa fa-times"></i> </a>
+          </li>
+        </ul>
+        <div class="menu-image menu-logo-mobile">
+          <a href="./index.html"> <img src="./images/logo.png" alt="Logo Image"> </a>
+        </div>
+      </div>
+      <nav class="navbar">
+        <div class="menu-image menu-image-desk">
+          <a href="./index.html"> <img src="./images/logo.png" alt="Logo Image"> </a>
+        </div>
+        <ul class="menu">
+          <li class="menu-item">
+            <a  class="menu-item-link" href="#home"> Home </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="#agendas"> Program </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="#speakers"> Speakers </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="./about.html"> About </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+  
+    <section id="home"  class="section-conference-invitation">
+      <!--<ul class="menu-bar">
+        <li>
+         <a href="#" class="hamburger">
+          <i class="fa fa-bars menu-bar-icon"></i>
+         </a>
+        </li>
+        <li>
+          <a href="#" class="close-icon"> <i class="fa fa-times"></i> </a>
+        </li>
+      </ul> -->
+      <div class="conference-description-container">
+        <h1 class="conference-name">4th International Conference on ICT for Development for Africa (2022) - (Upcoming)</h1>
+        <p class="conference-description">
+        The 4th International Conference on Information and Communication Technology for Development for Africa (ICT4DA2022), will take place in Bahir Dar, Ethiopia on 28-30 November, 2022. ICT4DA2022 is aims to bring together researchers, engineers, developers, and practitioners from the academia and the industry to present and discuss their research work in the area of ICT for Development.
+        </p>
+        <div class="date-and-place">
+          <span class="date"> 2022. 11.28. (Mon) ~ 30(Wed) </span>
+          <span class="place">Bahir Dar University, Bahir Dar, Ethiopia</span>      
+        </div>
+      </div>
+    </section>
+  
+  <!----Main program starts-->
+  <section id="agendas" class="main-program section-wrapper">
+    <h2 class="program-outline-topic section-title">Main Program</h2>
+    <div class="guide-bar"></div>
+    <div class="programs-wrapper">
```

Comment:
- [ ] Kindly fix the overflow on the mobile view whivh is causing the design to not be postioned properly

![image](https://user-images.githubusercontent.com/81932478/191733172-647b6491-fb14-486d-af47-41373d33a56a.png)
![image](https://user-images.githubusercontent.com/81932478/191733408-0711da5a-8595-42fe-971f-5543f65c79c4.png)

Reply:
Hi @Munsa1.
Thank you for your kind feedback. Can you explain more about the overflow of my design? My speakers section on my project differs from what you put the screenshot above?

Reply:
Hi @Munsa1. Thanks I added the footer section in the homepage mobile version within two lines in javascript, i.e., when the 'More' button is clicked, the entire speakers and the footer section is displayed.